backend/app/services/collections/create_collection.py

```
@@ -0,0 +1,205 @@
+                callback.fail(str(err))
+
+    except Exception as outer_err:
+        logger.error(
+            "[create_collection.execute_job] Unexpected Error during collection creation: %s",
+            str(outer_err),
+            exc_info=True,
+        )
+
+        collection_job.status = CollectionJobStatus.FAILED
+        collection_job.updated_at = now()
+        collection_job.error_message = str(err)
```

Ruff (0.13.3)

Undefined name `err`

(F821)